mmdet3d/core/bbox/structures/base_box3d.py

```diff
@@ -455,3 +455,18 @@ def new_box(self, data):
         original_type = type(self)
         return original_type(
             new_tensor, box_dim=self.box_dim, with_yaw=self.with_yaw)
+
+    def enlarged_box(self, extra_width):
```

Comment:
`enlarged_box` should not be defined here because of the difference between each coordinate.